### PR TITLE
Revert changes for Heapster to deploy it in Docker conatiner

### PR DESCRIPTION
**Pull request by zreigz**

### CODE REVIEW

File: build/hyperkube.sh [hunk 1]
```diff
@@ -43,3 +45,7 @@ docker run \
 
 docker run -d --net=host --privileged gcr.io/google_containers/hyperkube:v${K8S_VERSION} \
     /hyperkube proxy --master=http://127.0.0.1:${PORT} --v=2
+
+# Runs Heapster in standalone mode
+docker run --net=host -d kubernetes/heapster -port ${HEAPSTER_PORT} \
```
Comment (bryk) [inline build/hyperkube.sh]: Add heapster version here. E.g., `kubernetes/heapster:v6.7.8`.

File: build/cluster.js [hunk 2]
```diff
@@ -72,21 +66,15 @@ function clusterHealthCheck(doneFn) {
...
-        let podItems = podsStatus.items;
-        for (let pod of podItems) {
-          if (pod.hasOwnProperty("metadata")) {
-            if (pod.metadata.name.includes("heapster")) {
-              return doneFn(pod.status.phase);
-            }
-          }
-        }
+    let statistics = JSON.parse(stdout.trim());
+    Object.keys(statistics).forEach(function(key) {
```
Comment (bryk) [inline build/cluster.js]: How about just:
```js
let uptime = statistics.uptime;
```

File: build/cluster.js [hunk 3]
```diff
@@ -150,7 +124,7 @@ gulp.task('wait-for-heapster', function(doneFn) {
 
     // constantly query the heapster until it is properly running
     heapsterHealthCheck(function(result) {
-      if (result === runningState) {
+      if (!isNaN(result)) {
```
Comment (bryk) [inline build/cluster.js]: How about you call-back this function only when heapster is up and running. And do the `if (isFoo(result))` in the caller function.

**Comment by bryk** [comment]
Style comments. PTAL

File: build/hyperkube.sh (hunk 1 above)
Comment (zreigz) [inline build/hyperkube.sh]: done

File: build/cluster.js (hunk 2 above)
Comment (zreigz) [inline build/cluster.js]: done

File: build/cluster.js (hunk 3 above)
Comment (zreigz) [inline build/cluster.js]: done

**Comment by zreigz** [comment]
PTAL

**Comment by bryk** [comment]
LGTM. Will merge soon.

**Pull request merged**